hca-to-scea/test/golden/arguments.csv

```diff
@@ -1,12 +1,12 @@
 spreadsheet,HCA project uuid,study accession,E-HCAD accession,curator initials,experiment type,factor values,public release date,hca last update date
-SRP151260.xlsx,c893cb57-5c9f-4f26-9312-21b85be84313,SRP151260,50,AD,differential,disease,2009-02-27,2021-04-29
-SRP274475.xlsx,07073c12-8006-4710-a00b-23abdb814904,SRP274475,36,AD,differential,organ,2021-09-30,2020-11-25
-ERP121676_1.xlsx,955dfc2c-a8c6-4d04-aa4d-907610545d11,ERP121676,100,AD,baseline,individual,2021-09-30,2020-11-25
-ERP121676_2.xlsx,955dfc2c-a8c6-4d04-aa4d-907610545d11,ERP121676,100,AD,baseline,individual,2021-09-30,2020-11-25
-ERP121676_3.xlsx,955dfc2c-a8c6-4d04-aa4d-907610545d11,ERP121676,100,AD,baseline,individual,2021-09-30,2020-11-25
+#SRP151260.xlsx,c893cb57-5c9f-4f26-9312-21b85be84313,SRP151260,50,AD,differential,disease,2009-02-27,2021-04-29

```

Comment:
Why is this commented out?

Reply:
My plan is to keep these commented out in the pull requests specific to certain test cases, and then once the multiple pull requests have been merged into master, to create a new branch from master where I test these positive tests.

Reply:
(the other option would be to delete those lines instead of commenting them out)

Reply:
Ok, I'll delete the commented out part